src/cmds/qsub.c

```diff
@@ -164,7 +164,7 @@
 #define QSUB_DMN_TIMEOUT_LONG 60 /* timeout for qsub background process */
 #define QSUB_DMN_TIMEOUT_SHORT 5
 
-#define VOPT_ERROR 4321 /* Error code if -V option is detected in background qsub*/
+#define VOPT_SET 4 /* return code if -V option is detected in background qsub*/
```

Comment:
It is probably best to rename this return code to be more generic, so that it can be used later for other similar purposes, if necessary. Rename it to something like DMN_CANT_SERVE (or think something better)